workflow/comp_pred_ridge_and_LSTM.ipynb

```diff
@@ -34,20 +34,9 @@
   },
```

Comment:
<p>Small note: the user should comment out the line "mode = 'disabled'", otherwise it still doesn't work ;)</p><p> </p><p>Perhaps say 'and change <code>mode = 'disabled </code> to <code>mode = 'online'</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>